src/cli.ts

```diff
@@ -87,10 +87,8 @@ async function buildDirectory(
     const outChildPath = join(outDirPath, child);
     if ((await stat(srcChildPath)).isDirectory()) {
       await buildDirectory(srcChildPath, outChildPath, options);
-    } else if (srcChildPath.endsWith(extension)) {
-      const outPath = `${outChildPath.substr(0, outChildPath.length - extension.length)}.${
-        options.outExtension
-      }`;
+    } else if (extension.test(srcChildPath)) {
+      const outPath = _path.basename(outChildPath, _path.extname(outChildPath)) + '.' + options.outExtension;
```

Comment:
I'm getting `ReferenceError: _path is not defined` when I run this. Maybe you meant to import it?

Reply:
Fixed imports.